tests/integration/templates/firehose_kinesis_as_source.yaml

```diff
@@ -0,0 +1,47 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: test-bucket
+  stream:
+    Type: AWS::Kinesis::Stream
+    Properties: 
+      Name: testStream
+      ShardCount: 2    
+  deliveryStream:
+    DependsOn:
+      - bucket
+      - stream
+    Type: AWS::KinesisFirehose::DeliveryStream
+    Properties: 
+      DeliveryStreamName: deliveryStream
```

Comment:
Let's better use a randomized name, to avoid potential name clashes with other tests.

We can parameterize this name via:
```
       DeliveryStreamName: {{ stream_name }}
```
... and then use our `load_template(..)` util function to render the template with a random stream name:
```
stream_name = f"s-{short_uid()}"
load_template("firehose_kinesis_as_source.yaml", stream_name=stream_name)
```